src/test/java/com/thealgorithms/datastructures/graphs/WelshPowellAlgorithmTest.java

```diff
@@ -0,0 +1,63 @@
+package com.thealgorithms.datastructures.graphs;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import com.thealgorithms.datastructures.graphs.WelshPowell.Graph;
```

Comment:
This is not needed. One can use `var`. Please have a look below.
```suggestion
```

Reply:
i dont think i can remove it because then the isColoringValid method would  not work